### ISSUE TITLE
Bromley Council Loading Times

### ISSUE THREAD

**Issue by capandcamera** [mampfes/hacs_waste_collection_schedule#1269]
The bromley council source, does not always load correctly.
I have investigated this and it seems that there is some loading JS on the webpage which delays the page from completely loading.

This causes the integration to load nothing instead of the actual data.

The following selenium code seems to work as it waits for the specific elements to appear:

```

from selenium import webdriver
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.common.by import By
from selenium.webdriver.support import expected_conditions as EC

options = webdriver.ChromeOptions()
options.add_argument('--headless')
driver = webdriver.Chrome(options=options)
driver.get("https://recyclingservices.bromley.gov.uk/waste/6344676")
wait = WebDriverWait(driver, 20)
wait.until(EC.visibility_of_element_located((By.XPATH, "//dd[@class='waste__address__property']")))
html = driver.page_source
driver.close()

print(html)
```

**Comment by 5ila5** [comment]
I do not like the approach to execute java script it adds additional dependencies and may not behave consistently in different scenarios.
My implementation retries every 2 seconds as this is basically what the javascript on their website does anyway.

fixed by #1428 
fix will be part of release 1.44.0 or is available now as master (git) version (HACS: 3-dot menu -> re-download -> change version to master)

**Comment by capandcamera** [comment]
@5ila5 Thanks, Tested on Master and it seems to work well! Thanks again
